engine/src/test/java/io/camunda/zeebe/engine/util/ProcessingStateExtension.java

```diff
@@ -44,19 +44,19 @@
  * <ul>
  *   <li>{@code ZeebeDb} (exact type match)
  *   <li>{@code TransactionContext} (exact type match)
- *   <li>{@code MutableZeebeState} (exact type match or supertypes)
+ *   <li>{@code MutableprocessingState} (exact type match or supertypes)
  * </ul>
  *
  * on fields with the corresponding type.
  *
  * <p>Usage:
  *
  * <pre>{@code
- * @ExtendWith(ZeebeStateExtension)
+ * @ExtendWith(processingStateExtension)
  * public class Test {
  *   private ZeebeDb db; //will be injected
  *   private TransactionContext txContext; //will be injected
- *   private MutableZeebeState state //will be injected
+ *   private MutableprocessingState state //will be injected
```

Comment:
❌ These changes seem incorrect.

Reply:
Ugh, looks like I made a mess out of this :see_no_evil: 
I'll go through it and fix :+1:

Reply:
Haha, not at all. This is exactly what we have reviews for ❤️ I'm happy that you're doing the changes 🤗